<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>catalogEdit</class>
 <widget class="QDialog" name="catalogEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>362</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>362</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Редактировать товар</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Наименование товара:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineProduct">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Артикул</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineArt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePoints">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePrice">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSpecPrice">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Балл</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Спец. цена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCatAddApply">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
